RecoTauTag/Configuration/test/QCD-validation.txt

```diff
@@ -0,0 +1,5 @@
+/store/relval/CMSSW_8_1_0_pre11/RelValQCD_Pt_80_120_13/GEN-SIM-RECO/81X_mcRun2_asymptotic_Candidate_2016_08_30_11_31_55-v1/00000/24F576D4-4D74-E611-8898-0025905A48BA.root
```

Comment:
this seems inappropriate for 80X.
Please check and update to something based on 80X